something with asort is broken in gawk. It's not neccessary, so we just don't use it.

--- hwscan	2003-09-30 09:09:18.000000000 +0200
+++ hwscan	2004-01-28 17:16:38.000000000 +0100
@@ -321,7 +321,7 @@
 	get_isapnp_drivers();
 	get_usb_drivers();
 
-	asort(drivers);
+#asort(drivers);
 	for (i in drivers) {
 		id = drivers[i];
 		if ( driver_mod[id] && ! skip[id] ) {
